src/components/VideoDetails/VideoDetails.styles.js

```diff
@@ -83,13 +82,14 @@ export const FavoritesButton = styled.button`
   }
 
   &:hover {
-    background: #f44336;
-    color: white;
+    border-color: ${({ isThemeLight }) => (isThemeLight ? 'red' : 'white')};
+    background-color: ${({ isThemeLight }) => (isThemeLight ? 'red' : 'white')};
```

Comment:
Following my last comment her can reuse your code like:
```suggestion
    border-color: ${redOrWhite};
    background-color: ${redOrWhite};
```